packages/web-client/app/components/card-pay/deposit-workflow/transaction-amount/index.hbs

```diff
@@ -32,8 +32,13 @@
       >
       <div class="card-pay-transaction-amount__amount-field-input-container">
         {{svg-jar this.currentTokenDetails.icon class="card-pay-transaction-amount__amount-field-currency-icon" role="presentation"}}
+        {{!-- 
+          .boxel-input is a hack to get the boxel styles.  
+          depending on how we plan to handle validation of this input,
+          we may need to change Boxel's input to use a more DDAU pattern
+        --}}
         <input
```

Comment:
I'm not sure how we want to handle the input validation/filtering, but it doesn't seem easy to use Boxel's input component to do input filtering since Ember's `<Input>` component auto-updates its `@value`.

Reply:
This uses the `boxel-input` class directly to get styles from Boxel's input.